place_tracker/lib/place.dart

```diff
@@ -42,6 +42,27 @@ class Place {
       starRating: starRating ?? this.starRating,
     );
   }
+
+  @override
+  bool operator ==(Object other) =>
```

Comment:
Added equals/hashcode, so we can compare old to updated `Place` objects

Reply:
Is it worth migrating this to `freezed` or `built_value`?

Reply:
Definitely a good practice, until we get Dart data classes :smirk: